src/transformers/models/bark/generation_configuration_bark.py

```diff
@@ -86,6 +87,9 @@ def __init__(
                 Max length of semantic input vector.
             semantic_rate_hz (`float`, *optional*, defaults to 49.9):
                 Semantic rate in Hertz.
+            min_eos_p (`float`, *optional*):
+                Minimum threshold of the probability of the EOS token for it to be sampled. This is an early stopping
+                strategy to mitigate potential unwanted generations at the end of a prompt.
```

Comment:
Is there a default or a recommended value for bark? 

Reply:
By default `0.2`, do you think we should directly use default value ? it will change previous behaviour for sure!

Reply:
No, but we should mention it in the documentation !

Reply:
Yep, that's where I was trying to get 🤗